### ISSUE TITLE
Interosseous membrane calcification (leg)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10220]
**Preferred term label:**
Interosseous membrane calcification (leg)

**Synonyms**
Calcification between tibia and fibula

**Definition (free text, please give PubMed ID)**
This is similar to https://www.ncbi.nlm.nih.gov/medgen/C2674452 for which I requested an HPO term, but there is neither an HPO or UMLS term for the leg equivalent.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Osteogenesis imperfecta V, with calcification in interosseous membranes (https://omim.org/entry/610967)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000002
- label: Tbiofibular interosseous membrane calcification
- parents: HP:0100491
- pmid: PMID:10976985
- def: Deposition of excess calcium in the interosseous membrane of the leg, which extends between the interosseous crests of the tibia and fibula and separates the anterior and posterior muscles of the lower leg.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10220
- synonyms: Calcification of the middle tibiofibular joint(NONE)|Resistant hypertension(NONE)|Calcification of the interosseous membrane of leg(NONE)
